src/Aevatar.MongoDB/MongoDb/AevatarMongoDbModule.cs

```diff
@@ -25,12 +28,19 @@ public class AevatarMongoDbModule : AbpModule
 {
     public override void ConfigureServices(ServiceConfigurationContext context)
     {
+        
         //Example only, remove if not needed
         context.Services.AddMongoDbContext<AevatarMongoDbContext>(options =>
         {
             options.AddDefaultRepositories();
         });
 
+        context.Services.AddTransient<IApiKeysRepository, ApiKeyMongoRepository>();
+        context.Services.AddMongoDbContext<ApiKeysMongoDbContext>(options =>
```

Comment:
Maybe there's no need to create a ApiKeysMongoDbContext, AevatarMongoDbContext is enough?